addons/binding/org.openhab.binding.squeezebox/src/main/java/org/openhab/binding/squeezebox/internal/SqueezeBoxHandlerFactory.java

```diff
@@ -52,6 +59,9 @@
     private Map<ThingUID, ServiceRegistration<?>> discoveryServiceRegs = new HashMap<>();
 
     private AudioHTTPServer audioHTTPServer;
+    private NetworkAddressService networkAddressService;
+
+    private String callbackUrl;
```

Comment:
The NetworkAddressService will be set by
```
@Reference
    protected void setNetworkAddressService(NetworkAddressService networkAddressService) {
        this.networkAddressService = networkAddressService;
    }
```

and why should I remove the callback url? The binding needs the url, I just changed the way the binding retrieves the current IP address to create this Url.